src/nav/navReducer.js

```diff
@@ -63,7 +63,10 @@ const rehydrate = (state, action) => {
   if (rehydratedState.users === undefined) {
     return getStateForRoute('loading');
   }
-
+  // Dont switch to main UI if sharing screen is on.
```

Comment:
there's a reason for that blank line :wink: -- keep it there, and add a similar blank line before this new bit too

In general the basic principle here is: when making a small change to some existing code, notice the style it's written in and match the existing style.

The specific rationale for these blank lines is they help organize the code visually to match its logic.